### PR TITLE
Add Image namespace for Add-Ons service and Pod Infra Container Image

### PR DESCRIPTION
**Pull request by xiaoluhong**
add  Image namespace for  Add-Ons service  and Pod Infra Container Image

### CODE REVIEW

**Comment by xiaoluhong** [comment]
@yasker

**Comment by xiaoluhong** [comment]
issue：https://github.com/rancher/rancher/issues/9853

**Comment by deniseschannon** [comment]
@hongxiaolu @yasker This should be based on the `v1.6-development` branch and in the latest folders of k8s that haven't been released to the public (folder 33 and folder 34). We would never submit anything directly to `v1.6-release` branch without testing. 

Also, we wouldn't update any template that has already been released to the public. You'd need to start with a new folder.

File: infra-templates/k8s/32/rancher-compose.yml [hunk 1]
```diff
@@ -29,11 +29,23 @@
...
     type: boolean
   - variable: REGISTRY
-    description: The registry to pull addon images and the optional pod infra container image. This should be set to the private registry from which the images for addon-services (i.e. helm, heapster, dashboard, dns) and the pod infra container image should be pulled.
+    description: The registry to pull addon images and the optional pod infra container image. This should be set to the private registry from which the images for addon-services (i.e. helm, heapster, dashboard, dns) and the pod infra container image should be pulled ( Default：gcr.io ).
     label: Private Registry for Add-Ons and Pod Infra Container Image
     required: false
     default: ""
     type: string
+  - variable: IMAGE_NAMESPACE
+    description: images namespace for Add-Ons and Pod Infra Container Image ( Default：google_containers ).
```
Comment (alena1108) [inline infra-templates/k8s/32/rancher-compose.yml]: should default value be passed to the "default" field as well? for both IMAGE_NAMESPACE and KUBERNETES_HELM

File: infra-templates/k8s/32/rancher-compose.yml [hunk 2]
```diff
@@ -29,11 +29,23 @@
...
+    label: Image namespace for  Add-Ons and Pod Infra Container Image
+    required: false
+    default: ""
+    type: string
+  - variable: KUBERNETES_HELM
```
Comment (alena1108) [inline infra-templates/k8s/32/rancher-compose.yml]: as both parameters are namespaces, could we be consistent in naming things? Like HELM_IMAGE_NAMESPACE, BASE_IMAGE_NAMESPACE? Up to suggestions, but the names should reflect the meaning

File: infra-templates/k8s/32/rancher-compose.yml (hunk 1 above)
Comment (alena1108) [reply infra-templates/k8s/32/rancher-compose.yml]: I see that it is being defaulted on the kubernetes-package side, so ignore the comment.

File: infra-templates/k8s/32/rancher-compose.yml (hunk 2 above)
Comment (yasker) [reply infra-templates/k8s/32/rancher-compose.yml]: @hongxiaolu Let's do `HELM_IMAGE_NAMESPACE` and `BASE_IMAGE_NAMESPACE` like Alena suggested.

**Comment by alena1108** [comment]
LGTM

**Comment by yasker** [comment]
Just a reminder that please hold on for the merge since we haven't updated the k8s package image yet.

**Pull request closed without merging**